src/ray/protobuf/export_api/export_event.proto

```diff
@@ -47,5 +52,7 @@ message ExportEvent {
     ExportActorData actor_event_data = 6;
     ExportDriverJobEventData driver_job_event_data = 7;
     ExportSubmissionJobEventData submission_job_event_data = 8;
+    ExportTrainRun train_run = 9;
+    ExportTrainRunAttempt train_run_attempt = 10;
```

Comment:
Is there a naming convention for this? Should I include `Event` or `Data` or both as a suffix?

Reply:
@nikitavemuri Any preferences here?

Reply:
No strong preference but I think adding `EventData` would be most consistent. Ideally `ExportNodeData` and `ExportActorData` would also have `EventData`

Reply:
updated to have EventData suffix